wormchain/Dockerfile.proto

```diff
@@ -0,0 +1,18 @@
+FROM docker.io/golang:1.19.3@sha256:dc76ef03e54c34a00dcdca81e55c242d24b34d231637776c4bb5c1a8e8514253 AS go-build
```

Comment:
Why not use the `ignitehq/cli:0.23.0` docker image?  It comes with go installed and then we don't need to fetch the ignite cli separately.

Reply:
I wasn't able to run the following commands in the ignite docker image

Reply:
Thanks @jynnantonix. Using this Dockerfile